src/main/java/org/launchcode/techjobs/oo/JobField.java

```diff
@@ -0,0 +1,36 @@
+package org.launchcode.techjobs.oo;
+
+import java.util.Objects;
+
+public abstract class JobField {
+    private int id;
+    private static int nextId = 1;
+    private String value;
+
+    public JobField() {
+        id = nextId;
+        nextId++;
+    }
+    public JobField(String aValue){
+        this();
+        this.value = aValue;
+    }
+    public int getId() {
+        return id;
+    }
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+    @Override
+    public String toString() {
+        return value;
+    }
+    @Override
+    public int hashCode() {
+        return Objects.hash(getId());
+    }
+}
```

Comment:
```suggestion
    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (!(o instanceof JobField)) return false;
        JobField jobField = (JobField) o;
        return id == jobField.id;
    }
```
In JobField, you can define the custom` equals()` method to compare job fields by their IDs. You can use the Generate option and select ID to generate this method or copy one of `equals()` methods you created, paste and modify it to use `JobField` instead.